### PR TITLE
Fix Added plan information steps issue

### PR DESCRIPTION
**Pull request by paulopmt1**
task-related: https://github.com/Automattic/wp-calypso/pull/68058

Fix the bug found here https://github.com/Automattic/wp-calypso/pull/68177

### Test instructions

- After checkout this branch, open the new plugins plan page: [http://calypso.localhost:3000/plugins/woocommerce-subscriptions/mysimplesimplesite.wordpress.com?flags=plugins-plans-page](http://calypso.localhost:3000/plugins/woocommerce-subscriptions/mysimplesimplesite.wordpress.com?flags=plugins-plans-page) and make sure it works as expected:

	image

- Open "Tools" > "Earn" page and make sure it work as expected:

	image

### CODE REVIEW

File: client/my-sites/plugins/plans/index.tsx [hunk 1]
```diff
@@ -59,8 +61,34 @@ const Plans = ( { intervalType }: { intervalType: 'yearly' | 'monthly' } ) => {
...
+					"We guarantee site's performance and protect it from spammers detailing all activity records."
+				),
+				image: <Gridicon icon="plans" />,
+			},
+		],
+	};
+
 	return (
-		<MainComponent wideLayout>
+		<MainComponent className="plugin-plans" wideLayout>
```
Comment (paulopmt1) [inline client/my-sites/plugins/plans/index.tsx]: Added this main class to make sure we'll have the right scope

File: client/my-sites/plugins/plans/style.scss [hunk 2]
```diff
@@ -7,6 +9,67 @@
 	}
 }
 
+.plugin-plans .promo-section {
```
Comment (paulopmt1) [inline client/my-sites/plugins/plans/style.scss]: Added scoped CSS class

File: client/my-sites/plugins/plans/index.tsx (hunk 1 above)
Comment (DustyReagan) [reply client/my-sites/plugins/plans/index.tsx]: It looks like we also assign this `plugin-plans` class name to the ActionCard. How do you feel about renaming this to something like `plugins-plans-main` so it's more specific to the MainComponent?

Comment (paulopmt1) [reply client/my-sites/plugins/plans/index.tsx]: Sounds good @DustyReagan, I also improved the CSS a little bit, so now this information is aligned with the plans columns.

**Review by DustyReagan — APPROVED** [review]
Tested for regressions, looks good to me! 👍

**Review by lupus2k — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7548101

Thank you @paulopmt1 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7548101).

**Pull request merged**